### ISSUE TITLE
oh-my-bash: Shell is not interactive

### ISSUE THREAD

**Issue by AnderSouza** [ohmybash/oh-my-bash#293]
Hello there.
I've installed oh-my-bash and selected the powerline theme.
All good, everything worked fine.

When I restarted the system though, it was another matter. After logging in I received this error:

`
Error found when loading /home/[username]/.profile
oh-my-bash: Shell is not interactive.
As a result the session will not be configured correctly.
You should fix the problem as soon as feasible.
`

It turns out that I don't know how to fix the problem.
I have found a couple of issues on the web talking about the subject, 
but they don't seem to be directly related to oh-my-bash, or oh-my-zsh for that matter.

And then I ran into this issue: #18 .

Replacing the line 
```bash 
source $OSH/oh-my-bash.sh
```
with the following 

```bash
if tty -s
then
	source $OSH/oh-my-bash.sh
fi
```
solved my problem.

I initially thought that this problem had been resolved.
Is it possible that I did something wrong in the installation process?

I've created this issue to highlight the problem and the solution which, by the way, was already described in an old issue.
Hope it helps someone else.

**Comment by akinomyoga** [comment]
Thank you for the report. Hmm, OK. This is caused by 31f095f3ac7d9a6725b6646c35b131614833067f which was originally suggested in #73 https://github.com/ohmybash/oh-my-bash/pull/73/commits/f5c0018ccd5c9f76409758c995f294cd6e3c6203. I'll revert this commit.

**Comment by akinomyoga** [comment]
I have pushed a fix e5e3048. You can update oh-my-bash by running `upgrade_oh_my_bash`. Thank you

**Comment by AnderSouza** [comment]
Thank you.
